src/components/MainNav.tsx

```diff
@@ -73,6 +75,7 @@ export const MainNav = () => {
           <NavItem key={l.href} link={l} />
         ))}
       </div>
+      <ConnectWallet />
```

Comment:
We're missing the button on mobile. The `MainNav` component is only used on desktop. Both layouts are handled in `src/components/layout/LayoutHeader.tsx`.

![image](https://github.com/user-attachments/assets/5e47ef61-63d2-418c-a997-32ef44d354ff)

Reply:
how does our current theme switch work? wallets-kit has [buttonTheme](https://github.com/Creit-Tech/Stellar-Wallets-Kit/blob/bb96da9fa77f4a17e9b4c65826d5f52571cdbad3/src/stellar-wallets-kit.ts#L25C17-L25C29) that we can use.

Reply:
Nice! We can check the `data-sds-theme` attribute on `body`. 

![image](https://github.com/user-attachments/assets/9833d2da-6a2c-45da-bb57-8f9a68209b80)